server/socket/index.ts

```diff
@@ -82,18 +82,18 @@ export async function setupWebSocketServer(
   });
 
   // enables the admin analytics dashboard (creates /admin namespace)
-  instrument(io, {
-    auth: origin.includes('localhost')
-      ? false
-      : {
-          type: 'basic',
-          username: WEBSOCKET_ADMIN_USERNAME,
-          password: bcrypt.hashSync(
-            WEBSOCKET_ADMIN_PASSWORD,
-            WEBSOCKET_ADMIN_PASSWORD.length
-          ),
-        },
-  });
+  // instrument(io, {
```

Comment:
why is this commended out? should it just be deleted?

Reply:
It could be deleted. It provided us access to an admin dashboard for websockets which was not really used. The Bcrypt package used is incompatible with Mac M1 so we commented it out with Daniel while testing on his M1.